### PR TITLE
Readme update to include Fargate specific testing

### PR DESCRIPTION
**Pull request by shyam-amazon**
### What does this PR do?
Update the README.md content with Fargate profile specific testings

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/ssp-eks-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge? Yes

### Additional Notes

NA

### CODE REVIEW

**Review by kcoleman731 — APPROVED** [review]

File: examples/eks-cluster-with-fargate-profiles/README.md [hunk 1]
```diff
@@ -54,14 +54,19 @@ This following command used to update the `kubeconfig` in your local machine whe
...
+#### Step7: List all the nodes by running the command below and verify the fargate nodes
 
     $ kubectl get nodes
 
-#### Step7: List all the pods running in `kube-system` namespace
+#### Step8: List all the pods running in `kube-system` namespace
 
     $ kubectl get pods -n kube-system
 
+PS: CoreDNS requires additional setup if customers use only Fargate
```
Comment (askulkarni2) [inline examples/eks-cluster-with-fargate-profiles/README.md:69]: Change this to **Note**. Also you can please add reference to the doc that describes the additional setup required?

**Review by askulkarni2 — COMMENTED** [review]
@shyam-amazon Thank you for your PR, just one minor comment.

**Comment by vara-bonthu** [comment]
@shyam-amazon Could you please update the PR with @askulkarni2  comments? We can merge it once its updated

**Review by askulkarni2 — APPROVED** [review]

**Comment by askulkarni2** [comment]
@shyam-amazon please resolve merge conflicts and we can merge this then.

**Review by vara-bonthu — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**